src/scripts/Global.gd

```diff
@@ -117,6 +117,9 @@ var audio_stream: AudioStreamPlayer
 
 
 func _ready() -> void:
+	var window_size_multiplier := ceil(OS.get_screen_size().x / (2 * 640))
+	OS.window_size *= window_size_multiplier
```

Comment:
![Markipliers](https://user-images.githubusercontent.com/7307599/186795805-3d91dcd2-db0b-45ce-a826-f3710602195e.jpg)
??????????!!!!!!!!!!!!!

Reply:
our lord and savior